Marlin/src/feature/bedlevel/ubl/ubl.cpp

```diff
@@ -254,4 +255,81 @@ bool unified_bed_leveling::sanity_check() {
   return !!error_flag;
 }
 
+/**
+ * M1004: UBL Mesh Wizard - One-click mesh creation with or without a probe
+ */
+void GcodeSuite::M1004() {
+
+  #define ALIGN_GCODE TERN(Z_STEPPER_AUTO_ALIGN, "G34", "")
+  #define PROBE_GCODE TERN(HAS_BED_PROBE, "G29P1\nG29P3", "G29P4R255")
+
+  #if HAS_HOTEND // Handle H# parameter to set Hotend temp
+    celsius_t hotend_temp = 0;      
+    bool no_wait_for_cooling_hotend = false;
+    bool got_temp_hotend = false;
+
+    if (!got_temp_hotend) {
+      got_temp_hotend = parser.seenval('H');
+    if (got_temp_hotend) hotend_temp = parser.value_celsius();
+    }
+
+    if (!got_temp_hotend) {
+      no_wait_for_cooling_hotend = parser.seenval('H');
```

Comment:
`parser.seenval('H')` will always be `false` here because it's the same as the value of `got_temp_hotend`. So `no_wait_for_cooling_hotend` will never be set to `true`.

Reply:
bit of vestigial code that got reused can be removed.